### DISCUSSION TITLE
Creality K1 Max Bed Fans Issue

### DISCUSSION THREAD

**Discussion by CatkingEX** [Guilouz/Creality-Helper-Script-Wiki#692] [Issues]
I try following guide as below to get on the bed fan
--------------------------------------------------
PIN4 on the K1 board:
Add this as a 4th pin in printer.cfg(need root):

[output_pin fan4]
pin: PA0
pwm: True
cycle_time: 0.0100
hardware_pwm: false
value: 0.00
scale: 255
shutdown_value: 0.0
-----------------------------------------------------
it shows the fan control in the web console,
but it didn't get turn-on on the K1-MAX???
is the PA0 pin have no power supply or need another script???

plz help,
thanks a lot

**Comment by DuyKhongCay** [comment]
i think it should be PC0 pin. I have included the wrong pins
![3qo42gcq](https://github.com/user-attachments/assets/9301744d-2a35-44b7-b0f9-84c0eb2e18a2)
